documentation/sections/why_coffeescript.md

```diff
@@ -0,0 +1,43 @@
+## Why CoffeeScript?
+There are a few things you should know about CoffeeScript. 
+
+CoffeeScript is about: 
+
+  * Reducing complexity
+  * Keeping your code clean
+  * Being terse
+  * Removing as many 'Bad Parts' as possible
+  * Reducing code quality issues
+  * Increasing readability
+  * Being stable
+  * Bringing out the best from ECMAScript
+
+CoffeeScript is NOT about: 
+
+  * Having the latest XX/YY feature
+  * Implementing every aspect of ECMAScript, just because it's there.
+  * Trying to appease developers from other languages
+
+There are many style choices within the language, and they have been carefully thought out. Taking time out to understand how CoffeeScript works, and how to work well with it will help you write better, easier to read, safer and more reliable code.
+
+## Why not just use ECMAScript 7?
```

Comment:
The documentation always refers to ES5 or ES2015 or ES2015+, never “ES6” or “ES7”.